VAMobile/src/components/LoadingComponent/LoadingComponent.tsx

```diff
@@ -76,18 +79,13 @@ const LoadingComponent: FC<LoadingComponentProps> = ({
           {getSpinner()}
         </Box>
       ) : inlineSpinner ? (
-        <Box
```

Comment:
![image](https://github.com/department-of-veterans-affairs/va-mobile-app/assets/46909588/1ad0ab01-708c-4125-8dcd-92eaf5d3256b)
![Simulator Screenshot - iPhone 14 Pro - 2024-03-05 at 10 34 08](https://github.com/department-of-veterans-affairs/va-mobile-app/assets/46909588/a3cd39bb-b5ad-44bf-a0b8-78468f614bfe)
These are certainly being used in develop currently in messaging and look to just be avoiding a scrollview for some reason? We should deep dive why and how and if we can make edits to these or have to account for them.

Reply:
Just looked into this, and I seem to have missed this initially. The scroll view was being avoided because the default background color wasn't conforming to the background color used on the `ViewMessageScreen`. Instead of ignoring the `VAScrollView` and duplicating the code, I added a `scrollViewStyle` prop that gives the option to override the scroll view style and set the background color. This is what it looked like before/after for reference:

<img src="https://github.com/department-of-veterans-affairs/va-mobile-app/assets/9042882/0dfecff3-2451-4230-a28d-df96851e9e61" width="49%" />&nbsp;&nbsp;<img src="https://github.com/department-of-veterans-affairs/va-mobile-app/assets/9042882/e39ac03f-9d7e-4481-88e5-5de8ed18025a" width="49%" />